### ISSUE TITLE
support for stride method without arguments

### ISSUE THREAD

**Issue by skeydan** [mlverse/torch#106]
In Python:

```
t1 = torch.tensor([[1,2], [3,4], [5,6]])
t1

t1.stride()
```

In R we currently need to specify `dim`.

**Comment by skeydan** [comment]
obrigada :-)
